VERSION

```diff
@@ -1,2 +1,2 @@
-WAZUH-DOCKER_VERSION="3.11.4_7.6.1"
-REVISION="31140"
+WAZUH-DOCKER_VERSION="3.12.0_7.6.1"
+REVISION="31204"
```

Comment:
This should be `31200`

Reply:
Fixed in https://github.com/wazuh/wazuh-docker/pull/316/commits/cb60fc8b771a04b2c3beb652c305ffd5b316687e

Greetings !